leaf_size: 0.002
model_folder: ""
number_of_average_clouds: 3
number_of_median_clouds: 3

z_threshold: 0.02
planarSegmentationTolerance: 0.01
min_number_of_inliers: 10000
xmin: -0.5
xmax: 0.5
ymin: -0.5
ymax: 0.5
zmin: 0.5
zmax: 1.5

bin_size: 0.015
inlier_dist_segmentation: 0.01
segmentation_inlier_ratio: 0.20
max_number_of_instances: 4
max_fitness_score: 0.0003
inlier_dist_icp: 0.003
icp_inlier_threshold: 0.7
min_inlier_ratio_validation: 12
inlier_dist_validation: 0.0015

setNormalDistanceWeight: 0.11 #0.1
setDistanceThreshold: 0.03 #0.05
setRadiusLimits: 0.1


grid_res: 200
use_saved_pc: true
full_display: true
#save_clouds: true

# point_cloud_topic: /sr300/depth_registered/points
point_cloud_topic: /kinect2/hd/points
save_path: /home/hyoshdia/Documents/kinect2_pcl
